tests/NoResultQueryTest.php

```diff
@@ -76,7 +76,8 @@ public function testCreateTableAgainWillAddWarning()
 )';
 
         $connection->query($sql)->then(function (MysqlResult $command) {
-            $this->assertEquals(1, $command->warningCount);
+            // 3 warnings on MySQL 8+, 1 warning on legacy MySQL 5
+            $this->assertGreaterThanOrEqual(1, $command->warningCount);
```

Comment:
Not for this PR, but would it be worth it detecting the MySQL version and adjusting the number here based on that?

Reply:
Not opposed, but given the nature of this test, I don't think this has particularly high priority. PRs welcome I suppose :+1: